File: Java/Cuadrado.java
package Java;

public class Cuadrado extends Figura{
	private double lado;

	public Cuadrado(String nombre, double lado) {
		super(nombre);
		this.lado = lado;
	}

	public double getLado() {
		return lado;
	}
	
	public double getArea() {
		return lado * lado;
	}
	
	public String toString() {
		return "Cuadrado lado = " + lado;
	}
	
	
}

File: C++/Cuadrado.cc
#include "Cuadrado.h"
#include <iostream>

Cuadrado::Cuadrado(double lado){
  this->lado = lado;
}
double Cuadrado::area(){
  return lado * lado;
}

double Cuadrado::perimetro(){
  return 4 * lado;
}
